libs/styles/styles.css

```diff
@@ -2,6 +2,8 @@
  /*** Variables ***/
 
  :root {
+  --feds-totalheight-nav: calc(var(--feds-height-nav) + var(--feds-height-breadcrumbs));
```

Comment:
Just seeing this now (and I get that it was simply moved around for the purposes of this PR) and I'd advise caution when using this custom property. `--feds-height-nav` and `--feds-height-breadcrumbs` become available post-LCP, after the Gnav content has been fetched and a call to fetch the Gnav base styles has completed. So post-LCP + 2 calls.

Right now the value is used for setting `scroll-margin-top` for heading elements, which should not cause any issues, but using it for margins, paddings or other layout-changing properties could lead to increased CLS.

It might be safer to add some hardcoded fallbacks, in case someone decides to use this for other purposes: `--feds-totalheight-nav: calc(var(--feds-height-nav, 64px) + var(--feds-height-breadcrumbs, 33px));`. Or even go a step further and turn these fallback values into variables, since we already use them in a few places in this file.

Reply:
@overmyheadandbody I added some variables for the global navigation height and breadcrumb height values and added them as fallbacks on the `--feds-totalheight-nav` and also added the new variables to the places I thought appropriate in the file. Please let me know if you see any issues with my updates or have any other feedback. Thanks for the review!

Reply:
Thanks for implementing the changes, looking good 👍